AdrianSimionov/obsidian-mindmap-nextgen
# feat: implementation of inline markmap

**Issue #32 :** [link](https://github.com/AdrianSimionov/obsidian-mindmap-nextgen/issues/32)

### What has been done?

Implementation of inline markmap, which can be achieved by pasting or typing your markdown code inside a block:

````
```markmap
````

### How to test it?

1. Open obsidian or restart plugin
2. Begin to write anything
3. Create a` ```markmap ` code block
4. Verify the results
